kubeflow/core/kubeform_spawner.py

```diff
@@ -177,6 +177,26 @@ def _expand_user_properties(self, template):
 c.KubeSpawner.singleuser_working_dir = '/home/jovyan'
 volumes = []
 volume_mounts = []
+
+# Allow environment vars to override uid and gid.
```

Comment:
@pdmack Would you mind taking a look at this code? I remember it taking us a while to get uid and gid to work correctly when using a PV backing the home directory?

Reply:
By the way, I have been able to test that I can access the local directory when specified via the Jupyter Notebook and create/modify notebooks.
Also, if not specified, the notebook behaves as earlier. 
I haven't tested after creating the corresponding jsonnet params.

Reply:
Sorry, missed this. Merged but I'll take a closer look this weekend anyway.